airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/surrogate_key.sql

```diff
@@ -11,3 +11,12 @@
         {%- endfor %}
     )
 {%- endmacro %}
+
+{# MSSQL surrogate_key #}
+{%- macro sqlserver__surrogate_key(field_list) -%}
+    {% set fields = [] %}
+    {%- for field in field_list -%}
+        {% set _ = fields.append("coalesce(cast(" ~ field ~ " as " ~ dbt_utils.type_string() ~ "), '')") %}
+    {%- endfor -%}
+    {{dbt_utils.hash(dbt_utils.concat(fields))}}
+{%- endmacro -%}
```

Comment:
Does this differ from dbt_utils implementation of surrogate_key or you don't need to override it anymore now?

Reply:
I'll try to test it out, looks like it  has pretty same structure with `dbt_utils 0.6.4`

Reply:
It works, getting rid of `{%- macro sqlserver__surrogate_key(field_list) -%}` from `surrogate_key.sql`. Great catch!